src/scenes/game_view.cpp

```diff
@@ -36,15 +41,26 @@ class GameView : public Scene {
     EnumArray<PauseButton, bool> m_pause_button_pressed;
     bool m_should_draw_pause = false;
 
+    /* draggable items */
+    static constexpr std::size_t draggable_items_count = 3;
+    std::bitset<draggable_items_count> m_being_dragged;
+
     /* misc */
     std::vector<std::filesystem::path> m_card_image_paths;
+    bool m_show_chat = false;
+    bool m_something_dragged = false;
 
 protected:
     void on_window_attach() override {
         setup_background();
         setup_pause_menu();
         setup_hand();
         m_board.setup(m_window, &m_player_hand);
+        m_usernames_box.set_window(m_window);
+        for (int i = 1; i <= 4; i++) {
+            m_usernames_box.add_username(std::string("bebrik #") + std::to_string(i));
```

Comment:
kek, moi koresha used to call a shawarma place next to our Gerasimovskaya dormitory as "bebrik". It was the only place where we've eaten during our exams. In some sense bebrik saved our lives, otherwise moi koresha and me could drop out of university. God save bebrik.

UPD: it has been renamed to shawerland ([https://maps.app.goo.gl/nVnavp8LWKA95pH69](https://maps.app.goo.gl/nVnavp8LWKA95pH69)). damn